src/snowcli/cli/nativeapp/commands.py

```diff
@@ -0,0 +1,69 @@
+from typing import Optional
+import logging
+import typer
+
+from snowcli.cli.common.decorators import global_options
+from snowcli.cli.common.flags import DEFAULT_CONTEXT_SETTINGS
+from snowcli.output.decorators import with_output
+
+from .init import nativeapp_init
+from .manager import NativeAppManager
+
+from snowcli.output.types import (
+    CommandResult,
+    MessageResult,
+)
+
+app = typer.Typer(
+    context_settings=DEFAULT_CONTEXT_SETTINGS,
+    hidden=True,
+    name="app",
```

Comment:
One of discussions from our engineering review doc was about the name of this command: https://docs.google.com/document/d/1llIv8RTrvn4MA4brdh-mSOULegnjMm4nq-tWX3TUm0I/edit?disco=AAAAySr4GAI

Has the name of this command been approved by somebody? `snow app ...` is a very generic name.

<img width="365" alt="Screenshot 2023-09-11 at 12 55 01" src="https://github.com/Snowflake-Labs/snowcli/assets/132934591/b9cfccc9-286a-4b20-a6fb-ffdc22845c9b">

Reply:
Let's proceed with `app` but let's bring this topic on Thursday